composer.json

```diff
@@ -22,7 +22,7 @@
         "sonata-project/notification-bundle": "~2.2",
         "sonata-project/easy-extends-bundle": "~2.1",
         "sonata-project/doctrine-extensions": "~1.0",
-        "knplabs/gaufrette": ">=0.1.4",
+        "knplabs/gaufrette": ">=0.1.6",
```

Comment:
had a quick look into that library and found out why - it only started to support the php sdk 2.x with 0.1.6. In https://github.com/KnpLabs/Gaufrette/tree/v0.1.5/src/Gaufrette/Adapter and previous versions, you can see that the file AwsS3.php is missing - it was introduced in version 0.1.6